src/lib.rs

```diff
@@ -140,3 +210,4 @@ pub use bb8_postgres::tokio_postgres::tls::NoTls;
 #[cfg(feature = "asynk")]
 #[doc(hidden)]
 pub use async_trait::async_trait;
+use uuid::Uuid;
```

Comment:
this line should be moved to the top

Reply:
[fix spelling and move use Uuid to top](https://github.com/ayrat555/fang/pull/126/commits/5ef6ce7fc11addd07f404fc566e564f02532c1a3)